docs/install/first-run.md

```diff
@@ -84,7 +88,7 @@ chosen deployment model.
  - [Docker Stacks](./docker/stacks.md)
  - Kubernetes Stacks - *coming soon*
 
-### 2. Create a Project Template
+### 3. Create a Project Template
```

Comment:
Not needed for a first run?

Reply:
@knolleary @hardillb 

I guess we (I) should remove this section now? in this PR?

Reply:
I will be updating this section via #1250 to keep the changes together.